ui/components/app/alert-system/multiple-alert-modal/multiple-alert-modal.tsx

```diff
@@ -146,7 +146,22 @@ export function MultipleAlertModal({
   onFinalAcknowledgeClick,
   ownerId,
 }: MultipleAlertModalProps) {
-  const { isAlertConfirmed, alerts } = useAlerts(ownerId);
+  const { isAlertConfirmed, alerts: allAlerts } = useAlerts(ownerId);
+
+  // Remove duplicate warning alerts. Duplicates may occur
+  // when the same alert is shown for multiple fields.
+  const alerts = useMemo(() => {
+    return allAlerts.reduce((result: Alert[], alert: Alert) => {
+      const isDuplicateWarning = result.some((a: Alert) => {
+        return a.severity === Severity.Warning && a.message === alert.message;
```

Comment:
Currently, the only use case is for warning alerts (mismatch account). While this addresses the present scenario, should we extend it to handle duplicates for all alert severities?

Reply:
I agree. I think since the danger severity would require confirmation, it would require more updates. Because of this I think it’d be better to extend the functionality when we need it

Reply:
Assuming we definitely want duplicate alerts on different fields, would it be cleaner to encapsulate this within `useAlerts` in a property such as `uniqueAlerts`?